Control coverage/Feature/FrontDoor.md

```diff
@@ -0,0 +1,55 @@
+# Front Door
+
+**Resource Type:** Microsoft.Network/FrontDoors
+
+<!-- TOC -->
+
+- [Azure_FrontDoor_DP_Use_Secure_TLS_Version_Trial](#Azure_FrontDoor_DP_Use_Secure_TLS_Version_Trial)
+
+<!-- /TOC -->
+<br/>
+
+___ 
+
+## Azure_FrontDoor_DP_Use_Secure_TLS_Version_Trial
+
+### Display Name 
+[Trial] Front Door Classic should have Approved Minimum TLS version
+
+### Rationale 
+TLS provides privacy and data integrity between client and server. Using approved TLS version significantly reduces risks from security design issues and security bugs that may be present in older versions. 
+
+### Control Spec 
+
+> **Passed:** 
```

Comment:
If there are no custom domain then also control will be in passed state. Should we add that as well?

Reply:
Added